release-notes/v1.2.0.yaml

```diff
@@ -0,0 +1,149 @@
+  Fixed Ratelimit Deployment ignoring pod labels and annotation merge
+  Fixed the API Server receives unnecessary requests
+  Fixed terminating envoy pods don't respond with "Connection: close" (H1) or GOAWAY(H2) on shutdown, switch to an immediate drain strategy
+  Fixed ratelimit statsd not working
+  Fixed not generating selector of deployment/daemonset based on the custom label configuration of EnvoyProxy
+  Fixed egctl experimental translate using a wrong ns
+  Fixed reconcile not triggered for Secret updates referenced by a BackendTLSPolicy
+  Fixed Route with multiple parents has incorrect namespace in parentRef status
+  Fixed only log endpoint configuration in verbose logging mode (`-v 4` or higher)
+  Fixed the xDS translation failed when wasm http code source configured without a sha
+  Fixed HTTPRoute status only shows one parent when targeting multiple Gateways from different GatewayClasses
+  Fixed Route with multiple parents has incorrect namespace in parentRef status
+  Fixed BackendTlsPolicy specify multiple targetRefs of the same service, only one will work
+  Fixed Helm chart fails for Flux HelmRelease
+
+# Enhancements that improve performance.
+performance improvements: |
+  Fixed repeated resources and optimize memory usage
+
+# Other notable changes not covered by the above sections.
+Other changes: |
+  Upgraded Envoy to v1.32.1
+  Reduced the amount of configuration logging, and make it line-delimited friendly
+  Made watching alpha CRDs optional, so that gateway-api upgrade don't break envoy gateway
+  Removed grafana test framework from the addons helm chart
+  Disabled ALPN for non-HTTP routes
+  Added statPrefix for HCM and TCPProxy
+  Enabled GatewayHTTPListenerIsolation conformance test
+  Enabled GRPC conformance profile
+  Enabled HTTPRouteBackendRequestHeaderModifier conformance test
+  Added e2e test for Daemonset mode
+  Updated upgrades tests to use VERSION env variable
+  Fixed OVS scanner wrong license warnings
+  Added e2e test for Gateway with EnvoyProxy
+  Added e2e test for TLS session resumption
+  Added heap profile into benchmark report
+  Added e2e test for RecomputeRoute in ExtAuth
+  Added benchmark memory profiles into report
+  Fixed flaky gateway_with_conflicted_listener_cannot_be_merged e2e test
+  Fixed flaky Zipkin Tracing e2e test
+  Added e2e test for cookie based consistent hash load balancing
+  Added e2e test for load balancing
+  Fixed flaky authorization tests
+  Enabled upgrade test
+  Fixed flaky basic auth e2e test
+  Enabled use-client-protocol e2e test
+  Added performance benchmarking test for 1000 HTTPRoutes
+  Added e2e test for Datadog tracing
+  Added e2e tests for ratelimit invert matching headers
+  Reduced readinessProbe failureThreshold and periodSeconds
+  Bumped go-control-plane to v0.13.1
+  Enabled e2e tests for dual stack
+  Set ignore_health_on_host_removal to true for clusters with static endpoints
```

Comment:
can we move this to breaking changes with a note saying - this was done to speed up endpoint removal when active health check is configured.